cmd/demo/demo.go

```diff
@@ -25,21 +23,28 @@ var demoCmd = &cobra.Command{
 	Run: runDemo,
 }
 
-var testAPI bool
+var testAPIFlag bool
+var cfgFile, cfgNetFile string
 
 func init() {
```

Comment:
Thinking about it, we can probably put this whole function into `cmd/demo/Config.go`, (same for `root.go`) which would make it a bit easier to under stand since all the configuration is then in these two files.

Reply:
You did not like this :rofl: ?

Reply:
by the whole function you mean `func init()`? i thought init fits well in file demo.go, as the package is also called "demo" and hence this is where you might expect the init function to be located. furthermore, since init initializes the command line flags, i thought it made sense to keep them within that same file. however, i don't care too much. both ways (keep in demo.go or move to config.go) are fine for me. you decide.

Reply:
Then its ok.